### PR TITLE
Feat/dia 60 update theme and tokens

### PR DESCRIPTION
**Pull request by FrankieBortot**
Updated existing tokens, added new sematic tokens

### CODE REVIEW

File: src/themes/scripts/updateThemes.ts [hunk 1]
```diff
@@ -29,8 +29,16 @@ import { generateTheme } from '../utils/generateTheme'
 
 const getFile = (theme: string, file: string): string => resolve(THEMES_DIR, theme, file)
 
+const folderNameToIgnore = [
```
Comment (fredmaggiowski) [inline src/themes/scripts/updateThemes.ts:32]: Why is there a .DS_Store in the theme directory?

File: src/themes/utils/generateTheme.ts [hunk 2]
```diff
@@ -44,10 +44,10 @@ const resolveThemeValues = (themeValues: Theme, themeTokens: object) => (nodeVal
   const path = nodeValue.replace(PARENTHESES, '').split('.')
   const { $value } = get(themeValues, path) || {}
 
-  if (!$value) {
+  if ($value === undefined) {
```
Comment (fredmaggiowski) [inline src/themes/utils/generateTheme.ts:47]: why broading the check to allow for falsey values (e.g. null)? is this necessary to allow for `0`?

Comment (MIA-Deltat1995) [reply src/themes/utils/generateTheme.ts:47]: exactly

File: src/themes/scripts/updateThemes.ts (hunk 1 above)
Comment (MIA-Deltat1995) [reply src/themes/scripts/updateThemes.ts:32]: It is a folder inserted automatically by Finder of Mac OS. This is only a guard

**Review by GiovannaMonti — APPROVED** [review]

**Pull request merged**